exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Player.java

```diff
@@ -0,0 +1,61 @@
+package com.adaptionsoft.games.uglytrivia;
+
+import java.util.Random;
+import java.util.concurrent.Callable;
+
+public class Player {
+    private String name;
+    private int coin = 0;
+    private int placeOnBoard = 0;
+    private boolean isInPenalty = false;
+
+    private int diceValue;
+
+    public Player(String name) {
+        this.name = name;
+    }
+
+    String getName() {
+        return name;
+    }
+
+    int getPurse() {
+        return coin;
+    }
+
+    int getPlaceOnBoard() {
+        return placeOnBoard;
+    }
+
+    void addCoin() {
+        this.coin++;
+    }
+
+    void setPlaceOnBoard(int placeDisplacement) {
+        placeOnBoard += placeDisplacement;
+    }
+
+    void isGettingOutOfPenaltyBox(boolean gettingOut) {
+        if(gettingOut) {
+            System.out.println(name + " is getting out of the penalty box");
+        }
+        else {
+            System.out.println(name + " is not getting out of the penalty box");
+        }
+    }
+
+    void roll(String[] args) {
```

Comment:
You could use an Integer[] here instead of allowing the consumer to add strings. And or the value of the string could be verified before being passed into the Random constructor.